dffml/service/dev.py

```diff
@@ -756,6 +757,138 @@ async def run(self):
             self.logger.debug("Updated version file %s", version_file)
 
 
+class CommitLintError(Exception):
+    pass
+
+
+class LintCommits(CMD):
+    """
+    Enforce commit message style 
+    """
+
+    def _get_ignore_filter(self):
+        return lambda x: not x.endswith("_")
+
+    def _execute_func_list(self, param, func_list):
+        for func in func_list:
+            param = func(param)
+        return param
+
+    async def _get_file_mutations(self):
+        no_mutation = lambda x: x
+        mutation_func_factory = lambda ext: lambda x: x + ext
+        extensions = await self._get_all_exts()
+        mutations = {
+            "prefix_mutations": [
+                no_mutation,
+                lambda x: "." + x,
+                lambda x: "dffml/" + x,
+            ],
+            "suffix_mutations": [
+                no_mutation,
+                *[mutation_func_factory(ext) for ext in extensions],
+            ],
+        }
+        return mutations
+
+    async def _get_cmd_output(self, cmd: List[str]):
+        print(f"$ {' '.join(cmd)}")
+        proc = await asyncio.create_subprocess_exec(
+            *cmd, stdout=asyncio.subprocess.PIPE
```

Comment:
Let's make sure to run any `git` commands from the repo root, just in case someone is running commands from somewhere else.

```suggestion
            *cmd, stdout=asyncio.subprocess.PIPE, cwd=str(Path(__file__).parents[2])
```